packages/ses-ava/src/types.js

```diff
@@ -57,7 +57,7 @@
  * @param {ImplFunc} implFunc
  * @returns {void}
  *
- * @typedef {TesterFunc} TesterInterface
+ * @typedef {Object} TesterProperties
```

Comment:
Would it work with `@callback` ?

Reply:
Oddly enough, it seems we can have `@callback` with both `@param` and `@property` fields and that does seem to work.

Reply:
Ah, action at a distance strikes again. Using a callback pragma works locally but fails in static-module-record where I’m using `test.failing`.

```
Error: test/test-static-module-record.js(201,6): error TS2339: Property 'failing' does not exist on type 'TesterInterface'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reverting.

Reply:
Bummer. I remember @dckc running into issues with `@callback` too.